Nodes/ActuatorNode.cpp

```diff
@@ -44,7 +44,17 @@ void ActuatorNode::processMessage(const Message* message)
 	{
 		ActuatorPositionMsg* msg = (ActuatorPositionMsg*)message;
 
-		if(not MaestroController::writeCommand(MaestroCommands::SetPosition, m_Channel, msg->position()))
+		//Get relevant command
+		int setPosition;
+
+		if (nodeID() == NodeID::RudderActuator){
```

Comment:
If nodeID() returns a value that isn't one of these, setPosition will not be set and thus will be a random value which is then written to the node.